hazelcast-archunit-rules/src/main/java/com/hazelcast/test/archunit/ArchUnitRules.java

```diff
@@ -33,7 +33,9 @@
 import static com.tngtech.archunit.lang.conditions.ArchConditions.haveRawType;
 import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.classes;
 
-
+/**
+ * Common {@link ArchRule} for Hazelcast tests
+ */
```

Comment:
Does the rule also cover `com.hazelcast.test` package?

Reply:
Yes, `JavadocType`/`MissingJavadocType` is applied to everything, unless excluded specifically. The suppressions file is quite a mess now, I plan to clean it up.